### PR TITLE
Bump sql client libraries

### PR DESCRIPTION
**Pull request by muff1nman**

### CODE REVIEW

File: airsonic-main/pom.xml [hunk 1]
```diff
@@ -538,19 +538,19 @@
...
             <artifactId>postgresql</artifactId>
-            <version>42.1.4</version>
+            <version>42.2.5</version>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.43</version>
+            <version>8.0.16</version>
```
Comment (jvoisin) [inline airsonic-main/pom.xml]: Wow, skipping 2 majors version!

Comment (muff1nman) [reply airsonic-main/pom.xml]: Yeah I imagine this one is gonna require someone with mysql to test before it gets merged.

**Comment by eharris** [comment]
I have been considering converting over to mysql, but I'm currently hampered by the fact I don't want to enable networking for my DB instance, and the JDBC drivers on my platform don't support unix domain sockets.  I'll probably be upgrading that machine to a new release soon tho, hopefully that will provide a JDBC connector for mysql that will work with unix sockets.

**Comment by fxthomas** [comment]
I have no issues with `mysql-connector-java` 8.0.19 the latest MySQL 8 started with:

```
docker run --name mysql -e MYSQL_DATABASE=airsonic -e MYSQL_ROOT_PASSWORD=123456 -p 13306:3306 -d mysql:8.0 --character-set-server=utf8 --collation-server=utf8_general_ci
```

I think we can merge this.

**Comment by jvoisin** [comment]
Closed in favour of #1537

**Pull request closed without merging**